packages/react/lib/babel.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+module.exports = function({ types: t }) {
+  return {
+    visitor: {
+      CallExpression(path) {
+        if (path.node.callee.name !== 'Import') return;
```

Comment:
This is too fragile - we should check for the import path, to catch things where the function is being imported under a different name:
- `import { Import as LoadComponent } from '@untool/react';`
- or `const { Import: LoadComponent } = require('@untool/react');`
- or `const LoadComponent = require('@untool/react').Import;`

Reply:
I am not overly happy about it neither - do you have an idea how to find out the name of an imported/required function? Babel's docs are still a bit sparse...